Blackjack.py

```diff
@@ -0,0 +1,282 @@
+#This code is based on the Udemy course milestone project
+#This code will take the Class and Card classes to make a 52 card deck
+#Will ask the player to make a bet agains the dealer and then play blackjack
+#Player will be asek to hit or take another card
+#This is a bit unique as I forgot how to play Blackjack and so the program keeps playing
+#until all 52 cards are used up
+
+import random
+
+#These are my constants for the future I will UPPER_SNAKE_CASE
+suits = ('Hearts', 'Diamonds', 'Spades', 'Clubs')
+ranks = ('Two', 'Three', 'Four', 'Five', 'Six', 'Seven', 'Eight', 'Nine', 'Ten', 'Jack', 'Queen', 'King', 'Ace')
+values = {'Two':2, 'Three':3, 'Four':4, 'Five':5, 'Six':6, 'Seven':7, 'Eight':8, 'Nine':9, 'Ten':10, 'Jack':10,
+         'Queen':10, 'King':10, 'Ace':11}
+
+if __name__=="__main__":

```

Comment:
Almost, you want to leave your definitions above the __main__ check. That way they could be imported from another file. In this implementation they don't get defined unless it's run from the main file.

You only want your object creation and main loop to be inside here.

Reply:
fixed, thanks  for the tip